sources/osgShadow/ShadowMap.js

```diff
@@ -420,45 +420,45 @@ define( [
 
             this._shadowReceiveAttribute.setLightNumber( lightNumber );
 
-            this._receivingStateset.setAttributeAndModes( this._shadowReceiveAttribute, StateAttribute.ON | StateAttribute.OVERRIDE );
 
 
+            var fullOverride = StateAttribute.OVERRIDE | StateAttribute.ON;
```

Comment:
Idea is to make sure the null texture is "binded" to all shadow casting scene, so we override all existing texture bind in the scene, preventing any texture bind.

When user implements alpha mask casting, they use PROTECTED to prevent OVERRIDE to remove their alpha mask texture needed.